### PR TITLE
confirming file changes

### PR DESCRIPTION
**Pull request by zazie-codes**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [ x] I have read the contributing guide lines at https://docs.suricata.io/en/latest/devguide/codebase/contributing/contribution-process.html
- [ x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [x ] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:https://redmine.openinfosecfoundation.org/issues/6383

Describe changes:
- I added file description in all the listed C files excluding 'autoconf.h' and 'app-layer-dcerpc-common.h' as they do not exist.
- I added clear and accurate file description in each of the C files folowing the directed documentation standards.
- I assigned the author tag in 'app-layer-htp-libhtp.c', 'app-layer-htp-libhtp.h' and 'app-layer.h' 

### Provide values to any of the below to override the defaults.

To use a pull request use a branch name like `pr/N` where `N` is the
pull request number.

Alternatively, `SV_BRANCH` may also be a link to an
OISF/suricata-verify pull-request.

```
SV_REPO=
SV_BRANCH=
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=
```

### CODE REVIEW

**Review by victorjulien — CHANGES_REQUESTED** [review]
This commit contains many files that shouldn't have been part of it, like all the doxygen files. Please only add the intended files to the commit.

**Comment by zazie-codes** [comment]
> This commit contains many files that shouldn't have been part of it, like all the doxygen files. Please only add the intended files to the commit.

Sure let me correct that

File: src/app-layer-dnp3.h [hunk 1]
```diff
@@ -15,6 +15,11 @@
  * 02110-1301, USA.
  */
 
+/**
+ *\file
+ *
+ * This file contains DNP3  Protocol Handling
```
Comment (jufajardini) [inline src/app-layer-dnp3.h:21]: Maybe...
`DNP3 application layer protocol header file`?

**Review by jufajardini — CHANGES_REQUESTED** [review]
Also, please make sure to adjust the commit message to follow our commit guidelines (https://docs.suricata.io/en/latest/devguide/codebase/contributing/code-submission-process.html#commits), and to incorporate the feedback we're giving into a new PR .

I've left one inline comment with a style suggestion for the documentation comment. :)

**Pull request closed without merging**